admin.php

```diff
@@ -7,39 +7,124 @@
+
+    /**
+     * Template folder directory.
+     */
+    const TEMPLATE_DIR = __DIR__ . '/tpl';
+
+    const TEMPLATE_FILE = 'admin.tpl';
+
+    public function getMenuSort()
+    {
+        return self::MENU_SORT;
     }
 
     /**
-     * @return bool true if only access for superuser, false is for superusers and moderators
+     * Admins or higher can access this plugin.
+     *
+     * You do not need to be a superuser to access this plugin.
      */
-    public function forAdminOnly() {
+    public function forAdminOnly()
+    {
         return true;
     }
 
     /**
-     * Should carry out any processing required by the plugin.
+     * HTML output (gets generated by twig).
      */
-    public function handle() {
+    public function html()
+    {
+        global $conf;
```

Comment:
Ugh, global. But if this is how Dokuwiki does things I guess it's fine.

Reply:
Well to be honest, this global isn't even used, so I will gladly remove it :sweat_smile:.